### PR TITLE
Add support for pang13

### PR DESCRIPTION
**Pull request by leviport**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.80~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.80
```
Comment (jackpot51) [inline debian/changelog:3]: After this line, you need a line `* Add pang13`

Comment (jackpot51) [reply debian/changelog:3]: Fixed

**Review by jackpot51 — APPROVED** [review]

File: debian/changelog (hunk 1 above)
Comment (leviport) [reply debian/changelog:3]: Ah, spaced that. Thanks for fixing

**Review by jacobktm — APPROVED** [review]

**Comment by jackpot51** [comment]
This needs a make-release is not already done

**Pull request merged**